monitoring/README.md

```diff
@@ -178,11 +178,11 @@ Filters can be provided as a sequence of `,`-separated values. Right-most filter
 
 | name                               | severity | description                                          |
 | ---                                | ---      | ---                                                  |
-| `certificate.stake.registration`   | `debug`  | A new stake credential registration was processed    |
-| `certificate.stake.delegation`     | `debug`  | A new stake delegation was processed                 |
-| `certificate.stake.deregistration` | `debug`  | A new stake credential de-registration was processed |
-| `certificate.pool.registration`    | `debug`  | A new stake pool registration was processed          |
-| `certificate.pool.retirement`      | `debug`  | A new stake pool retirement was processed            |
+| `certificate.stake.registration`   | `trace`  | A new stake credential registration was processed    |
```

Comment:
@KtorZ we should investigate ways to generate those docs to ensure they don't go out of sync

Reply:
Cool! I will keep that in mind!